choreographer/src/main/java/eu/arrowhead/core/choreographer/service/ChoreographerService.java

```diff
@@ -99,6 +115,14 @@ public void init() {
         requesterSystem.setSystemName(registrationProperties.getCoreSystemName().toLowerCase());
         requesterSystem.setAddress(registrationProperties.getCoreSystemDomainName());
         requesterSystem.setPort(registrationProperties.getCoreSystemDomainPort());
+        
+        if (sslProperties.isSslEnabled()) {
+        	ownCloud = new CloudRequestDTO();
+            final String serverCN = (String) arrowheadContext.get(CommonConstants.SERVER_COMMON_NAME);
+            final String[] serverFields = serverCN.split("\\.");
+            ownCloud.setName(serverFields[1]);
+            ownCloud.setOperator(serverFields[2]);
+        }
```

Comment:
this is unnecessary I think. If I recall correctly, the authorization token generation does the same, if no cloud is provided